sormas-backend/src/test/java/de/symeda/sormas/backend/ArchitectureTest.java

```diff
@@ -102,8 +144,22 @@ private void assertFacadeEjbAnnotated(Class<?> facadeEjbClass, boolean classAuth
 			methods.should()
 				.beAnnotatedWith(RolesAllowed.class)
 				.orShould()
-				.haveNameMatching("^(get|count|is|does|has|validate|to|pseudonymize|convertToReferenceDto|fillOrBuild|convertToDto|fromDto).*")
+				.haveNameMatching(
+					"^(get|count|is|does|has|validate|to|pseudonymize|convertToReferenceDto|fillOrBuild|convertToDto|fromDto|exists"
+						+ getAdditionalNameMatchingStringForFacade(facadeEjbClass) + ").*")
 				.check(classes);
 		}
 	}
+
+	private String getAdditionalNameMatchingStringForFacade(Class<?> facadeEjbClass) {
```

Comment:
Using this solution can lead to a situation where the developers put all kind of methods here that they don't know how to annotatate.
for `LabMessageFacadeEjb` the test method should be changed to test allow testing that there is an annotation on the classOnly with some exceptions(the ones that are for _SYSTEM)